src/server/config/schema.js

```diff
@@ -31,8 +31,15 @@ module.exports = Joi.object({
     defaultRoute: Joi.string(),
     ssl: Joi.object({
       cert: Joi.string(),
-      key: Joi.string()
-    }).default(),
+      key: Joi.string(),
+      enabled: Joi.boolean().when('cert', {
```

Comment:
Shouldn't there be some sort of similar assertion behavior tied to this for `key` as well?